snooty/postprocess.py

```diff
@@ -204,6 +205,40 @@ def run_event_parser(
     def reset_program(self, filename: FileId, page: Page) -> None:
         self.pending_program = None
 
+    def _attach_doc_title(self, filename: FileId, node: n.RefRole) -> None:
+        if not isinstance(node.fileid, str):
+            line = node.span[0]
+            self.diagnostics[filename].append(
+                Diagnostic.error(f'File not specified: "{node.name}"', line)
+            )
+            return
+
+        # Add .txt to end of doc role target path
+        target_path = PurePath(node.fileid)
```

Comment:
Hm. A couple questions:

* Why are we instantiating a PurePath rather than keeping it as a FileId?
* Do we need to append the `.txt` suffix, since that's not actually used in the lookup afaics?

Reply:
Good point! I did this because I wasn't sure if `reroot_path()` would work properly without the extensions, but... it does! Thanks 🎉